.travis.yml

```diff
@@ -26,22 +26,22 @@ env:
 matrix:
   include:
     - php: 7.1
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=PGSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_TEST=1
+      env: DB=PGSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_COVERAGE_TEST=1
     - php: 7.2
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev BEHAT_TEST=1
-    - php: 7.2
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1
-    - php: 7.2
-      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev NPM_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev BEHAT_TEST=1
     - php: 7.2
       env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1
     - php: 7.3
-      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPCS_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.6.x-dev NPM_TEST=1
     - php: 7.3
+      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1
+    - php: 7.4
+      env: DB=MYSQL RECIPE_VERSION=4.6.x-dev PHPCS_TEST=1
```

Comment:
You don't need two PHPCS tests.

Reply:
Correction, there's three now, there were two originally.

Reply:
```suggestion
      env: DB=MYSQL RECIPE_VERSION=4.6.x-dev
```